docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,150 @@
+---
+title: Branch Review
+sidebar_position: 35
+---
+
+Cypress Branch Review is designed to elevate your pull request (PR) review workflow. It allows you to quickly identify the impact a pull request might have on your test suite in a single view. Compare which tests are failing, flaky, pending, added, or modified between the **<u>source</u>** and **<u>base</u>** branches.
+
+Previously, pinpointing changes in your test suite's results required a manual side-by-side comparison between your newly-introduced branch's test runs and your base branch's test runs. This sub-optimal workflow often fails to answer the fundamental questions, _what changed_ and _why?_ You might be left wondering if the same tests are flaky between branches, when new failures were introduced, or if you added sufficient test coverage to your new branch.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review3.jpg"
+  alt="Cypress Branch Review"
+/>
+
+:::info
+
+Cypress Branch Review is currently only available for GitHub integrations. We are working on adding support for GitLab and Bitbucket soon.
+
+:::
+
+## Getting Started
+
+:::tip
+
+To access the Branch Review public beta, an admin can simply enable it under "Cypress Labs" in your Organization Settings in Cypress Cloud.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/labs_beta.png"
+  alt="Enable Branch Review Beta"
+/>
+
+:::
+
+Your project will first need to be connected to a GitHub repository. If you haven't already, follow the steps in [Install the Cypress GitHub app](/guides/cloud/integrations/source-control/github#Install-the-Cypress-GitHub-app) to connect your project to GitHub.
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have an open pull request.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/get-to-branch-review.jpg"
+  alt="Get to Branch Review"
+/>
```

Comment:
Maybe use an image with PR tags on the branches? We may want to link to docs (or briefly mention) about sending PR numbers to runs.

Reply:
Meaning this: https://docs.cypress.io/guides/continuous-integration/introduction#CI-Build-Information

Reply:
I'll look to get this in a state where PR tags are represented.